Library/macOSRichTextEditor/Source/RichTextEditor.m

```diff
@@ -1052,6 +1060,57 @@ - (NSFont *)fontwithBoldTrait:(NSNumber *)isBold italicTrait:(NSNumber *)isItali
 	return newFont;
 }
 
+/**
+ * Does not allow cursor to be right beside the bullet point.
+ *
+ * @param beginRange The beginning position of the paragraph
+ * @param currentRange The current cursor position
+ */
+- (NSRange)moveCursorAroundBulletListIfApplicableWithBeginningLocation:(NSRange)beginRange andPrevious:(NSRange)previousRange andCurrent:(NSRange)currentRange isMouseClick:(BOOL)mouse {
```

Comment:
Let's rename this to `adjustSelectedRangeForBulletsWithStart:Previous:andCurrent:isMouseClick:`.

Your documentation is also out of date. Move the part about this moving the selected range "if applicable" to the documentation.